e2e/test.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+
+describe('InstantSearch e-commerce demo', () => {
+  before(() => {
+    browser.maximizeWindow();
+  });
+
+  it('must work', async () => {
+    await browser.url(
+      'https://deploy-preview-3967--instantsearchjs.netlify.com/examples/e-commerce/'
+    );
+
+    // Select "Apple" brand in list
+    const brand = await browser.$('span=Apple');
+    await brand.click();
+
+    // Fill search input with "macbook"
+    const searchInput = await browser.$('[type=search]');
+    await searchInput.setValue('macbook');
+
+    // Wait for the results list to be updated (wait for the "macbook" word to be highlighted)
+    await browser.waitUntil(
+      async () => (await browser.$$('mark=MacBook')).length > 0,
+      10000
```

Comment:
can we not rely on the search callback instead of a magic number?

Reply:
What do you mean by search callback? You can find the API documentation here https://webdriver.io/docs/api.html

Reply:
We can wait for the `result` event of `search` for example. However that isn't very portable between frameworks. Since we know there's only one search done, the search client could be injected & waited for maybe

Reply:
Since it is an E2E test we're not supposed to interact with InstantSearch differently from a user.

We should only wait for a visual feedback that the search is done, like a real user.